src/dynamodb.js

```diff
@@ -26,21 +26,41 @@ function buildConfigFromConnection (connection) {
 }
 
 async function createTables (dynamoClient, uniqueIdentifier) {
-  await Promise.all(tablesSchema.map(async table => {
-    const newTable = cloneDeep(table);
-    const TableName = getTableName(newTable.TableName, uniqueIdentifier);
-    newTable.TableName = TableName;
-    await dynamoClient.createTable(newTable).promise();
-    await dynamoClient.waitFor('tableExists', { TableName });
-  }));
+  const provisionedTables = [];
+  await Promise.all(
+    tablesSchema.map(async table => {
+      const newTable = cloneDeep(table);
+      const TableName = getTableName(newTable.TableName, uniqueIdentifier);
+      newTable.TableName = TableName;
+
+      try {
+        await dynamoClient.createTable(newTable).promise();
+        await dynamoClient.waitFor('tableExists', { TableName }).promise();
+        provisionedTables.push(TableName);
+      } catch (err) {
+        console.error(`Failed to create table "${TableName}"`, err);
+      }
+    })
+  );
+  return provisionedTables;
```

Comment:
How can the caller tell if there was an error? Should you include the error in the response?